scene/gui/color_picker.cpp

```diff
@@ -1986,13 +2164,23 @@ ColorPicker::ColorPicker() {
 	wheel->set_mouse_filter(MOUSE_FILTER_PASS);
 	wheel->connect(SceneStringName(draw), callable_mp(this, &ColorPicker::_hsv_draw).bind(2, wheel));
 
+	// HACK: I cannot draw focus stylebox on the wheel itself, as it's drawing based on shader.
+	wheel_h_focus_display = memnew(Control);
+	wheel_margin->add_child(wheel_h_focus_display);
+	wheel_h_focus_display->set_mouse_filter(MOUSE_FILTER_PASS);
+	wheel_h_focus_display->set_focus_mode(FOCUS_ALL);
+	wheel_h_focus_display->connect(SceneStringName(draw), callable_mp(this, &ColorPicker::_hsv_draw).bind(3, wheel_h_focus_display));
+	wheel_h_focus_display->connect(SceneStringName(gui_input), callable_mp(this, &ColorPicker::_uv_input).bind(wheel_h_focus_display));
```

Comment:
We should be able to draw a special circle focus stylebox (using a dedicated StyleBoxFlat in the editor theme/default project theme). This can be left for a future PR as it's not essential.

Reply:
I modified the theme to have separate styleboxes - one for rectangle-shaped elements and another for circle-shaped elements:
![obraz](https://github.com/user-attachments/assets/97bc2f63-30af-41a5-bd71-c44553c63d8e)

![obraz](https://github.com/user-attachments/assets/9ca0e886-8f6a-4a52-8e44-e73593092c51)